tests/contributing.py

```diff
@@ -121,3 +124,65 @@ def validate_transformer_integration(transformer):
     print(tabulate(summary, headers='keys', showindex=False))
 
     return validation_error is None and error_trace is None
+
+
+def validate_transformer_quality(transformer):
+    """Validate quality tests for a transformer.
+
+    This function creates a DataFrame containing the results
+    from running the quality tests for this transformer against
+    all the datasets with columns of its input type. It does the
+    following steps:
+    1. A DataFrame containing the regression scores obtained from running the
+    transformers of the input type against the datasets in the test cases is
+    created. Each row in the DataFrame has the transformer name, dataset name,
+    column name and score. The scores are computed as follows:
+        - For every transformer of the data type, transform all the
+        columns of that data type.
+        - For every numerical column in the dataset, the transformed
+        columns are used as features to train a regression model.
+        - The score is the coefficient of determination obtained from
+        that model trying to predict the target column.
+    2. Once the scores are gathered, a results table is created. Each row has
+    a transformer name, dataset name, average score for the dataset,
+    a score comparing the transformer's average score for the dataset to
+    the average of the average score for the dataset across all transformers of
+    the same data type, and whether or not the score passed the test threshold.
+    3. The table described above is printed when this function is run.
+
+    Args:
+        transformer (string or rdt.transformers.BaseTransformer):
+            The transformer to validate.
+    Output:
+        bool:
```

Comment:
It seems like this is not describing the actual output.

Reply:
Good catch!